book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] Good job so far, but you should have a method for adding rentals and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Please kindly add it 🙃 😉 
It can be similar to 👇🏾 

```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

- [ ] And the `@rentals` is `Getters and Setters` not only a `Getter`, it should be an `attr_accessor`. Kindly fix that 🙏🏾